portfolio/src/main/webapp/index.html

```diff
@@ -124,9 +124,17 @@ <h2>Technical Skills</h2>
 	<section class="sectionRow" id="comments">
 		<h2 class="sectionHeader">Comments</h2>
 		<hr class="round">
-		<!--<div id="commentNumberSelector">
-			<button id= "deleteCommentsButton" class="button" onclick="deleteComments()">Delete All Comments</button>     
-		</div> -->
+		<div id="commentNumberSelector">
+			<label>Language: </label>
+			<select id = "selectedLanguage" onchange="loadComments()">
```

Comment:
Nit: Maybe remove spaces in the id attribute so it matches the rest of this file